portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/UserMgtClientServiceImpl.java

```diff
@@ -37,6 +38,9 @@
     private static final String USERNAME_CLAIM = "http://wso2.org/claims/username";
     private static final String GROUPNAME_CLAIM = "http://wso2.org/claims/groupname";
```

Comment:
Can we use `groupName` here ? @Pushpalanka I think you used the same format when adding the group attribute to the